### ISSUE TITLE
Can not reproduce the search result on PTB dataset (Fig3)?

### ISSUE THREAD

**Issue by fortunechen** [quark0/darts#155]
Hi, everyone.

I try to reproduce the model search performance in Fig 3 on PTB dataset. Becauce it tells us as searching time growing, the searched model will be improved. 

![Fig3](https://user-images.githubusercontent.com/26025440/108598429-73e56900-73c8-11eb-9d9e-fd6c964ff487.png)

 I run the `train_search.py` code  4 times with different random seed. Then I get the result below. The x-axis denotes how many gpu hours I trained to get the model and y-axis denotes the performance of searched model after training for 300 epoch.  Except for neccssacy changes to run the code successfully on `pytorch1.7 +cuda 11.`. I didn't change any code in original code.

The result shows below, we can see that as searching time grows, the searched model will **NOT**  be improved. 

![my reproduce](https://user-images.githubusercontent.com/26025440/108598217-5368df00-73c7-11eb-8182-a6f055602881.png)

**Comment by fortunechen** [comment]
Here is the code I used [darts-rnn](https://github.com/fortunechen/mydarts-rnn) with pytorch1.7+cuda11.0

**Comment by fortunechen** [comment]
I will try reproduce the result on `pytorch 0.3.1` again.

**Comment by chaoji90** [comment]
Empirically, darts search time and performance are often inversely proportional and the early stop is usually inevitable.

**Comment by fortunechen** [comment]
> Empirically, darts search time and performance are often inversely proportional and the early stop is usually inevitable.

I agree so. However the total search time is the same as the paper(50 epochs). and the preformace is much lower than the author's.

**Comment by fortunechen** [comment]
> I will try reproduce the result on `pytorch 0.3.1` again.

Here is the result without any modification of original code.

![result_origin](https://user-images.githubusercontent.com/26025440/108960202-7ee21700-76b0-11eb-9b86-f91640b07200.png)
